providers/default.rb

```diff
@@ -56,6 +56,7 @@ def whyrun_supported?
     user node[:druid][:user]
     group node[:druid][:group]
     only_if { ::Dir.glob(druid_archive).empty? }
```

Comment:
does this mean that bumping up git ref won't trigger rebuild? how did we upgrade druid previously?